testing/trino-product-tests/src/main/java/io/trino/tests/product/hudi/TestHudiSparkCompatibility.java

```diff
@@ -320,6 +319,21 @@ public void testTimelineTableRedirect()
         }
     }
 
+    @Test(groups = {HUDI, PROFILE_SPECIFIC_TESTS})
+    public void testReadTableWithReplaceCommits()
+    {
+        String tableName = "test_hudi_cow_replace_commits_select_" + randomNameSuffix();
+
+        createNonPartitionedTableWithReplaceCommit(tableName, COW_TABLE_TYPE);
+
+        try {
+            onTrino().executeQuery("SELECT id, name FROM hudi.default." + tableName);
```

Comment:
Please verify the SELECT result.

Reply:
Done - what I want to see here though is that the query passes. If it does, then replace commits were parsed in the timeline successfully. Please let me know if you thought about some additional verification here.

Reply:
Please verify the rows using `containsOnly` instead of `hasAnyRows`.